docs/rule/no-unnecessary-curly-strings.md

```diff
@@ -0,0 +1,29 @@
+# no-unnecessary-curly-strings
+
+Strings need not be wrapped in the mustache expressions.
```

Comment:
```suggestion
Strings need not be wrapped in curly braces.
```

Maybe? Since the rule name uses `curly`, and we also use `curly braces` in the error message.

Reply:
Updated mention both.